Docs - Sync Security LDAP guide with QuickStart & improve (#28479)
quarkusio/quarkus: docs/src/main/asciidoc/security-ldap.adoc

----
quarkus.security.ldap.enabled=true

quarkus.security.ldap.dir-context.principal=uid=admin,ou=system
quarkus.security.ldap.dir-context.url=ldaps://ldap.server.local <1>
quarkus.security.ldap.dir-context.password=secret

quarkus.security.ldap.identity-mapping.rdn-identifier=uid
quarkus.security.ldap.identity-mapping.search-base-dn=ou=Users,dc=quarkus,dc=io

quarkus.security.ldap.identity-mapping.attribute-mappings."0".from=cn
quarkus.security.ldap.identity-mapping.attribute-mappings."0".filter=(member=uid={0},ou=Users,dc=quarkus,dc=io) <2>
quarkus.security.ldap.identity-mapping.attribute-mappings."0".filter-base-dn=ou=Roles,dc=quarkus,dc=io

%test.quarkus.security.ldap.dir-context.url=ldap://127.0.0.1:10389 <3>
----

> <1> You need to provide the URL to an LDAP server. This example requires the LDAP server that has imported [following LDIF file](https://github.com/quarkusio/quarkus/blob/main/test-framework/ldap/src/main/resources/quarkus-io.ldif).

Reviewer: ```suggestion
<1> You need to provide the URL to an LDAP server. This example requires the LDAP server to have imported [this LDIF file](https://github.com/quarkusio/quarkus/blob/main/test-framework/ldap/src/main/resources/quarkus-io.ldif).
```

<2> `{0}` is substituted by the `uid`.

> <3> The URL used by our test resource. Tests may leverage `LdapServerTestResource` provided by Quarkus as [we do](https://github.com/quarkusio/quarkus-quickstarts/blob/main/security-ldap-quickstart/src/test/java/org/acme/elytron/security/ldap/ElytronLdapExtensionTestResources.java) in the test coverage of the example application.

Reviewer: Be careful, here and above you are using Markdown links and not Asciidoc links. it should be `https://the.link[the text]`.

Author: Ah, I messed it up. Will fix it.

Author: Fixed, thank you
